CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing to CTS
+
+Thanks for your interest in Consul-Terraform-Sync. We value feedback and contributions from the community whether it is a bug report, enhancements, new features or documentation.
+
+Please read through this document before submitting any issues or pull requests to ensure we have all the necessary 
+information to effectively respond to your bug report or contribution.
+
+
+## Reporting Issues, Bugs or Feature Requests
+
+We welcome CTS users to report issues or suggest features.
+
+If you're suggesting a new feature (i.e., functionality that doesn't exist yet), please use our [issue template](https://github.com/hashicorp/consul-terraform-sync/issues/new?assignees=&labels=&template=feature_request.md).  This will prompt you to answer a few questions that will help us figure out what you're looking for.  The template will also tag incoming issues with "Enhancement".  This gives us a way to filter the community-opened issues quickly so we can review as a team.
+
+Check for duplicates when filing an issue. Please check [existing open](https://github.com/hashicorp/consul-terraform-sync/issues), or [recently closed](https://github.com/hashicorp/consul-terraform-sync/issues?q=is%3Aissue+is%3Aclosed) issues to make sure somebody else hasn't already reported the issue. 
+
+
+If you're reporting what you think is a bug (i.e., something isn't right with an existing feature), please try to include as much information as you can. Details like these are incredibly useful:
+
+* A reproducible test case or series of steps performed
+* The version of our code being used for our products(CTS version, Consul version, Terraform version, Terraform provider(s) version, etc.)
+* Any modifications you've made relevant to the bug
+* Anything unusual about your environment or deployment
```

Comment:
```suggestion
* Anything unique about your environment or deployment that may be useful to consider for debugging
```

Minor suggestion to change "unusual" to "unique"; the former seems to ask the operator to call themselves out on bad practices they have deployed.